RegexFileSearcher/RegexFileSearcher/FileHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
```

Comment:
```suggestion
using System.Diagnostics;
using System.Runtime.InteropServices;
```
Unnecessary `using` directives.